framework/src/outputs/Console.C

```diff
@@ -97,12 +97,12 @@ InputParameters validParams<Console>()
   params.set<MultiMooseEnum>("execute_system_information_on", /*quiet_mode=*/true) = "initial";
 
   // Change the default behavior of 'execute_on' to included nonlinear iterations and failed timesteps
-  params.set<MultiMooseEnum>("execute_on", /*quiet_mode=*/true).push_back("timestep_begin linear nonlinear failed");
+  params.set<MultiMooseEnum>("execute_on", /*quiet_mode=*/true).push_back("initial timestep_begin linear nonlinear failed");
 
   // By default postprocessors and scalar are only output at the end of a timestep
-  params.set<MultiMooseEnum>("execute_postprocessors_on", /*quiet_mode=*/true) = "timestep_end";
-  params.set<MultiMooseEnum>("execute_vector_postprocessors_on", /*quiet_mode=*/true) = "timestep_end";
-  params.set<MultiMooseEnum>("execute_scalars_on", /*quiet_mode=*/true) = "timestep_end";
+  params.set<MultiMooseEnum>("execute_postprocessors_on", /*quiet_mode=*/true) = "initial timestep_end";
+  params.set<MultiMooseEnum>("execute_vector_postprocessors_on", /*quiet_mode=*/true) = "initial timestep_end";
+  params.set<MultiMooseEnum>("execute_scalars_on", /*quiet_mode=*/true) = "initial timestep_end";
```

Comment:
Oh, don't open that can of worms again. That used to be the case, put then
when you changed the execute on the top level it feeds down and turns off
your postprocessors tables and people get mad, Derek and I went around and
around on this. The final solution is the Console output is "special":
https://github.com/idaholab/moose/issues/4497

On Thu, Sep 10, 2015 at 4:16 PM, Cody Permann <notifications@github.com>
wrote:

> In framework/src/outputs/Console.C
> <https://github.com/idaholab/moose/pull/5682#discussion_r39221496>:
>
> >
> >    // By default postprocessors and scalar are only output at the end of a timestep
> > -  params.set<MultiMooseEnum>("execute_postprocessors_on", /*quiet_mode=*/true) = "timestep_end";
> > -  params.set<MultiMooseEnum>("execute_vector_postprocessors_on", /*quiet_mode=*/true) = "timestep_end";
> > -  params.set<MultiMooseEnum>("execute_scalars_on", /*quiet_mode=*/true) = "timestep_end";
> > +  params.set<MultiMooseEnum>("execute_postprocessors_on", /*quiet_mode=*/true) = "initial timestep_end";
> > +  params.set<MultiMooseEnum>("execute_vector_postprocessors_on", /*quiet_mode=*/true) = "initial timestep_end";
> > +  params.set<MultiMooseEnum>("execute_scalars_on", /*quiet_mode=*/true) = "initial timestep_end";
>
> Maybe this has already been discussed but wouldn't a sensible default be
> to output the postprocessors whenever they are executed (i.e. match the
> execution time?) Otherwise if you adjust one but not the other you may
> experience difficult to understand behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5682/files#r39221496>.
>